app/move/controllers/create/court-hearings.js

```diff
@@ -43,43 +42,37 @@ class CourtHearingsController extends CreateBaseController {
     }
   }
 
-  async saveValues(req, res, next) {
+  saveValues(req, res, next) {
     // TODO: Remove once we support creating hearings without a case
     if (req.form.values.has_court_case === 'false') {
-      return next()
+      return super.saveValues(req, res, next)
     }
 
-    const moveDate = req.sessionModel.get('date')
     const {
       court_hearing__comments: comments,
       court_hearing__court_case: courtCaseId,
       court_hearing__start_time: startTime,
     } = req.form.values
-    const courtCase = find(req.courtCases, {
-      id: courtCaseId,
-    })
+    const moveDate = req.sessionModel.get('date')
+    const courtCase = find(req.courtCases, { id: courtCaseId })
+    const hearingDatetime = parseISO(`${moveDate}T${startTime}`)
```

Comment:
Is this really the only place we do (or will ever do) this? Just seems like a crucial bit of formatting is a bit buried away here

Reply:
Valid point. It's the only time right now where we ask the user for only a time, and convert it to a datetime. 

It's also quite specific here where we give it the move date (from an earlier step) and use the time entered on this step so felt logical to do it here.